src/views/Exam.vue

```diff
@@ -138,23 +138,19 @@ export default {
     },
     apollo: {
         userQuiz: {
-            query: UserQuizzesQuery,
+            query: UserQuizQuery,
+            variables() {
+                return { quizID: this.$route.params.quizId }
+            },
             update: (data) => {
-                return data.userQuizzes[0]
+                return data.userQuiz
             },
+            fetchPolicy: 'no-cache',
```

Comment:
There is an issue with apollo cache @Will-ZYS  please have a look @scorpionknifes  gave up. Remove line to remove hot fix.

Reply:
fixed